torch/_torch_docs.py

```diff
@@ -68,7 +69,7 @@ def parse_kwargs(desc):
         returned tensor. Default: ``False``.
 """)
 
-add_docstr(torch.abs,
+add_docstr(_C.abs,
```

Comment:
What does this fix? torch.* and _C.* functions are exactly the same object so precious code should be fine